integration/indexes_compat_test.go

```diff
@@ -311,179 +311,6 @@ func TestCreateIndexesCompat(t *testing.T) {
 	}
 }
 
-// TestCreateIndexesCommandCompat tests specific behavior for index creation that can be only provided through RunCommand.
-func TestCreateIndexesCommandCompat(t *testing.T) {
```

Comment:
Moved to `indexes_command_compat_test.go` according to the testing naming convention.